README.md

```diff
@@ -87,29 +87,35 @@ Summary of one-time project setup:
 $ source venv/bin/activate
 ```
 
-2. Find the test file that contains the test you want to run.
+2. Check the `tests/unit_tests` folder, and find the test file you want to run
 
-   - Check the `tests` folder, and find the test file you want to run
    - In that test file, read through each test case
+      - If it is incomplete, complete the test. **There are incomplete tests for Waves 01, 02, and 06**. 
+        - *Is this a nominal or edge case?*
+        - *What type of input do we need to test this case?*
+        - *What is the expected output for the given input?*
+   - Remove the lines that contain `@pytest.mark.skip()` for the test(s) you want to run.
```

Comment:
It doesn't seem like `@pytest.mark.skip()` is used for any tests in this project, so could be omitted unless this is just general information included for all projects

Reply:
Oh! Good catch. I was thinking I'd add these to each test. Though open to leaving them off. I know they are in viewing party for each test.

Reply:
If students won't have seen this before, would it be worth adding something that reminds them to search the project/test files for `@pytest.mark.skip()` to ensure no tests were missed before submitting the assignment?

Reply:
Added @pytest.mark.skip to all tests and added a note to search for this in the readme (actually viewing-party readme, which this readme now points to)